### ISSUE TITLE
Missing team permission to delete repos

### ISSUE THREAD

**Issue by silverwind** [go-gitea/gitea#32436]
### Feature Description

I would like to grant the ability to delete repos to a team, but even with all write-related permissions granted, the users are unable to see the repo "Danger Zone".

I think a new boolean option "Delete repositories" needs to be added to the "Repository access" section in the screenshot, which shows the team permission configuration.

### Screenshots

<img width="1181" alt="image" src="https://github.com/user-attachments/assets/486c8d29-3d46-4412-8515-dce60c111e3a">

**Comment by lunny** [comment]
Administration permission is the answer

**Comment by silverwind** [comment]
Ok, then I think the text "Members can pull and push to team repositories and add collaborators to them.
" should be updated to mention that it also grants access to "dangerous actions" like repo deletion.

**Comment by silverwind** [comment]
> user can always create issues no matter they have read or write permission. So write only means edit here. but for other units, write also contains the meaning of creating.

Yeah. I also noticed that a user with "read" access to issues can create issue comments, so you are correct.

**Comment by wxiaoguang** [comment]
Maybe related to Members cannot delete repositories they have created on their own #38058

The quick fix is: #39063
